client/src/app/tabs/cloud-dmn/__tests__/DmnEditorSpec.js

```diff
@@ -1320,34 +1318,37 @@ describe('<DmnEditor>', function() {
 
       instance.handleChanged();
 
-      wrapper.update();
-
-      const toggle = wrapper.find('#button-toggle-overview');
-
       // when
-      toggle.simulate('click');
+      const toggle = document.querySelector('#button-toggle-overview');
+      fireEvent.click(toggle);
 
       // then
-      expect(layout.dmnOverview.open).to.be.true;
+      await waitFor(() => {
+        expect(onLayoutChanged).to.have.been.calledOnce;
+      });
+
+      const callArg = onLayoutChanged.getCall(0).args[0];
```

Comment:
I personally prefer the assertion within the spy, to prevent this and the following line :)

Reply:
Done